docs/install.md

```diff
@@ -1,9 +1,19 @@
 # Install guide
 
-You can install Presidio as a service in [Kubernetes](https://kubernetes.io/) or use it as a framework
+You can install Presidio locally using [Docker](), as a service in [Kubernetes](https://kubernetes.io/) or use it as a framework in a python application.
 
 ## The easy way with Docker
 
+You will need to have Docker installed and running, and [make](https://www.gnu.org/software/make/) make installed on your system.
+
+Sync this repo use `make` to build and deploy locally.
+
+For convenience the script `build.sh` at the root of this repo will run the `make` commands for you. If you use the script remember to make it executable by running `chmod +x build.sh` after syncing the code.
+
+**NOTE: Building the deps images currently takes some time** (~70 minutes, depending on the build machine). We are working on improving the build time through improving the build and providing pre-built dependencies.
```

Comment:
Should we add an additional note about the errors which get printed out when running this without a container registry?